temporalio/exceptions.py

```diff
@@ -109,6 +112,16 @@ def non_retryable(self) -> bool:
         """
         return self._non_retryable
 
+    @property
+    def next_retry_delay(self) -> Optional[timedelta]:
+        """Delay before the next activity retry attempt.
+
+        User activity code may set this when raising ApplicationError to specify
+        a delay before the next activity retry. Ignored if set when raising
+        ApplicationError from workflow code.
```

Comment:
I don't believe the last sentence is true anymore: https://github.com/temporalio/api/pull/426